lib/ansible/modules/cloud/ovirt/ovirt_vms.py

```diff
@@ -66,6 +66,14 @@
             - "Specify if latest template version should be used, when running a stateless VM."
             - "If this parameter is set to I(true) stateless VM is created."
         version_added: "2.3"
+    storage_domain:
+        description:
+            - "Name of the storage domain where all template disks should be copied."
+            - "This parameter is considered only when template is provided."
+    disk_format:
+        description:
+            - "COW or RAW; default: is COW"
```

Comment:
Please remove this line and replace with

```
choices: ['COW, 'RAW']
default: COW
```


See http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#documentation-block for more info